src/components/Kv/KvFlag.vue

```diff
@@ -0,0 +1,1038 @@
+<template>
+	<div :class="`flag-icon ${country.toLowerCase()}`"></div>
+</template>
+
+<script>
+export default {
+	props: {
+		country: { type: String, required: true }
+	},
+};
+</script>
+
+<style lang="scss" scoped>
+	.flag-icon {
+		background: url('../../assets/images/flags.png') no-repeat;
```

Comment:
We have `@` built-in to our webpack config as a shortcut to the `src` directory, so the url can be written as `url('~@/assets/images/flags.png')` (ex: [fonts.scss](https://github.com/kiva/ui/blob/25cb1aedd602678ab7eddf485e4765b55f7e919c/src/assets/scss/global/fonts.scss#L4)). Doing that will make it so that we don't have to update this line if we move this component around.

Reply:
Oh awesome, thank you for showing me that 👍 just updated my PR